README.md

```diff
@@ -1,14 +1,16 @@
 # Samba-Gluster Integration Test Environment
```

Comment:
I think we should remove the "Gluster" part.

Reply:
nice catch! updated!